tests/integration/inventories/openvswitch.yaml

```diff
@@ -0,0 +1,16 @@
+---
+all:
+  hosts:
+    localhost:
+      ansible_connection: local
+      ansible_python_interpreter: /usr/bin/python3.8
+appliance:
+  hosts:
+    openvswitch:
+      ansible_host: openvswitch.example.org
+      ansible_user: zuul
+  vars:
+    ansible_connection: ssh
+    collection_name: openvswitch
+    collection_namespace: openvswitch
+    prepare_ovs_tests_task: False
```

Comment:
lets remove this, I'd prefer not to leak testing configuration into the inventory file if possible.

If we want to do this, lets just update the collection to stop doing it.